### PR TITLE
ENG-2798 implement bundle undeploy feature

### PR DESCRIPTION
**Pull request by ichalagashvili**

### CODE REVIEW

File: src/state/component-repository/hub/actions.js [hunk 1]
```diff
@@ -260,17 +261,24 @@ export const sendDeployBundle = bundle => (dispatch, getState) => (
...
         if (response.ok) {
           dispatch(addToast(
             { id: 'app.undeployed', values: { type: 'bundle', code: bundle.name } },
             TOAST_SUCCESS,
           ));
-          // @TODO do same as above when deploying is done
+          dispatch(fetchSelectedBundleStatus(bundle.gitRepoAddress || bundle.componentUrl));
+          const state = getState();
+          const components = getECRComponentList(state);
+          dispatch(setECRComponents([...components.filter(c => c.code !== bundle.componentCode)]));
```
Comment (jeffgo10) [inline src/state/component-repository/hub/actions.js]: not sure why you still need to array spread this part. seems redundant

**Review by jeffgo10 — CHANGES_REQUESTED** [review]
just one concern

File: src/state/component-repository/hub/actions.js (hunk 1 above)
Comment (ichalagashvili) [reply src/state/component-repository/hub/actions.js]: ah right. let me change it

**Review by jeffgo10 — APPROVED** [review]
ayt 👍

**Pull request merged**